lib/jnpr/junos/utils/sw.py

```diff
@@ -70,6 +70,8 @@ def __init__(self, dev):
             self._multi_RE is True and dev.facts.get('vc_capable') is True and
             dev.facts.get('vc_mode') != 'Disabled')
         self._mixed_VC = bool(dev.facts.get('vc_mode') == 'Mixed')
+        self._single_re_issu = bool('current_re' in dev.facts and
+                                    'localre' in dev.facts['current_re'])
```

Comment:
```
{master:0}
user@s0> file show /etc/hosts.junos | match localre 
128.0.0.63       localre
```
The platforms which support single-RE ISSU use this internal IP address for communication during the ISSU process.